.github/workflows/keda-build-pull.yaml

```diff
@@ -0,0 +1,16 @@
+name: keda build (pull)
+
+on:
+  pull_request_target:
+    types: [opened, edited, synchronize, reopened, ready_for_review]
```

Comment:
Do we need build for docs?
Now I see difference between pull and push build:
```
    paths-ignore:
      - "docs/**"
      - "examples/**"
```

Reply:
Also in test-infra we have pre-keda-manager-operator-build job with skip_if_only_changed_documentation.